resources/assets/js/app/views/messages/messageDialogItem.js

```diff
@@ -19,6 +24,32 @@ module.exports = Marionette.ItemView.extend({
     clickedEditMessage: function () {
         Radio.channel('messagesChannel').trigger('editMessage', this.model);
     },
+    mouseenterMsgFrom: function (e) {
+        var intervalMinutes = App.getConfigAttr('messageChangeOnDelay');
+        var intervalMilliseconds = intervalMinutes * 60 * 1000;
+        var now = new Date();
+        var nowUTC = new Date(
+            now.getUTCFullYear(),
+            now.getUTCMonth(),
+            now.getUTCDate(),
+            now.getUTCHours(),
+            now.getUTCMinutes(),
+            now.getUTCSeconds()
+        );
```

Comment:
Please look if you can use some of helper function for your needs here: https://github.com/BinaryStudioAcademy/bsa-2016-forum/blob/next/resources/assets/js/app/helpers/dateHelper.js

if no any suitable function, then move your functionality for time calculation to helper function